### ISSUE TITLE
Sniper not working...

### ISSUE THREAD

**Issue by WaffleMan0310** [NECROBOTIO/NecroBot#2833]
I have enabled the sniping ability of the bot, but it has yet to actually snipe anything. I have made sure all of the right settings are enabled and I believe I did it right. I remember when using the bot previously, if I went above the threshold for pokeballs, it would pop up a message saying that it was scanning for snipable pokemon and then go proceed to bring the goods. Now I see nothing of the sort.

"UseSnipeOnlineLocationServer": true,
  "UseSnipeLocationServer": true,
  "SnipeLocationServer": "localhost",
  "SnipeLocationServerPort": 16969,
  "GetSniperInfoFromPokezz": true,
  "GetOnlyVerifiedSniperInfoFromPokezz": false,
  "MinPokeballsToSnipe": 80,
  "MinPokeballsWhileSnipe": 0,
  "MinDelayBetweenSnipes": 5000,
  "SnipingScanOffset": 0.003,
  "SnipeAtPokestops": true,
  "SnipeIgnoreUnknownIv": true,
  "UseTransferIvForSnipe": false,
  "SnipePokemonNotInPokedex": true,

**Comment by relaxingtomato** [comment]
I have the same problem.

**Comment by Bobguy123** [comment]
Same problem on my end

**Comment by ChemiHatake** [comment]
"SnipeIgnoreUnknownIv" if the IV of the pokemon is unknown, it won't snipe it.
try turning that off.

**Comment by aapikz** [comment]
same issue.. pogo feeder is working fine from pokesniper and trackmon.. coordintes is perfectly showing up in debug.. 
but necrobot doesn't snipe anything.. it just collecting pokestop and not sniping

**Comment by itz420anytime** [comment]
turn the regular use snipe location off then try it

**Comment by itz420anytime** [comment]
keep the online one true

**Comment by Linewalker** [comment]
Snipe locations with a negative long, than it might be this: https://github.com/NECROBOTIO/NecroBot/issues/2822

**Comment by MaverickTse** [comment]
Works with Pokezz:
`
"UseSnipeOnlineLocationServer": true,
  "UseSnipeLocationServer": false,
  "SnipeLocationServer": "localhost",
  "SnipeLocationServerPort": 16969,
  "GetSniperInfoFromPokezz": true,
  "GetOnlyVerifiedSniperInfoFromPokezz": true,
"SnipingScanOffset": 0.003,
  "SnipeAtPokestops": true,
  "SnipeIgnoreUnknownIv": false,
  "UseTransferIvForSnipe": false,
  "SnipePokemonNotInPokedex": false,
`
basically just set "SnipeAtPokestops" to true.
![necrobot_snipezz](https://cloud.githubusercontent.com/assets/10569988/17463625/63c96b3a-5cfc-11e6-9c08-fed814693ab8.PNG)

**Comment by aapikz** [comment]
when you set UseSnipeLocationServer": false 
it will use lat, long, you setup in config right?
 "PokemonToSnipe": {
    "Locations": [
      {
        "Latitude": 38.556807486461118,
        "Longitude": -121.2383794784546
      },
      {
        "Latitude": -33.859019,
        "Longitude": 151.213098
      },
      {
        "Latitude": 47.5014969,
        "Longitude": -122.0959568
      },
      {
        "Latitude": 51.5025343,
        "Longitude": -0.2055027
      }
    ],


it just sniping my lat,long setup... so how can i make it snipe from pokezz?

**Comment by WaffleMan0310** [comment]
I always thought it would snipe from the location server at localhost, port 16969 if you set "UseSnipeLocationServer":true. But I might be wrong...

**Comment by madnessxd** [comment]
@aapikz use UseOnlineSnipeLocationServer for pokezz etc. (Also make sure that GetSniperInfoFromPokezz is set to true.)
use UseSnipeLocationServer for your own setup.

**Comment by WaffleMan0310** [comment]
With the following settings it scans the lat and longs specified in the sniping filter section at the bottom of the config.

"UseSnipeOnlineLocationServer": true,
  "UseSnipeLocationServer": false,
  "SnipeLocationServer": "localhost",
  "SnipeLocationServerPort": 16969,
  "GetSniperInfoFromPokezz": true,
  "GetOnlyVerifiedSniperInfoFromPokezz": true,
  "MinPokeballsToSnipe": 80,
  "MinPokeballsWhileSnipe": 0,
  "MinDelayBetweenSnipes": 5000,
  "SnipingScanOffset": 0.003,
  "SnipeAtPokestops": true,
  "SnipeIgnoreUnknownIv": false,
  "UseTransferIvForSnipe": false,
  "SnipePokemonNotInPokedex": true,

But it does not appear to use an online server, despite it being specified above.

**Comment by aapikz** [comment]
would guys share your config? we cant really make auto sniping work

**Comment by WaffleMan0310** [comment]
http://pastebin.com/trS4ZJdf

**Comment by madnessxd** [comment]
This is my setup:
GetOnlyVerifiedSniperInfoFromPokezz is not mandatory

  "UseSnipeOnlineLocationServer": true,
  "UseSnipeLocationServer": false,
  "SnipeLocationServer": "localhost",
  "SnipeLocationServerPort": 16969,
  "GetSniperInfoFromPokezz": true,
  "GetOnlyVerifiedSniperInfoFromPokezz": false,
  "MinPokeballsToSnipe": 20,
  "MinPokeballsWhileSnipe": 0,
  "MinDelayBetweenSnipes": 60000,
  "SnipingScanOffset": 0.003,
  "SnipeAtPokestops": true,
  "SnipeIgnoreUnknownIv": false,
  "UseTransferIvForSnipe": false,
  "SnipePokemonNotInPokedex": false,

  "UsePokemonSniperFilterOnly": false,

**Comment by aapikz** [comment]
i have the same snipe config.. but i cant still make it work.. 
can you share your whole config.json... i think it might because of bot circulation..

**Comment by madnessxd** [comment]
http://pastebin.com/bSEk8Cqa

**Comment by WaffleMan0310** [comment]
Anyone have a clue as to why it is not using the online server when "UseSnipeOnlineLocationServer" is true?

**Comment by aapikz** [comment]
madnessxd i tried your config. but i think your bot is for manual sniping.. not auto sniping from pokezz.com
it says looking for snipable pokemon at -37.7651298,145.3509665
no pokemon found... 
then colect pokestop. 
rinse and repeat..

auto sniping is must be like this 
"sniping dragonite at-37.812183 , 144.960644"

**Comment by madnessxd** [comment]
@aapikz He does both for some reason. I just caught a Raichu, Gyarados and Flareon just 5 minutes ago. He starts with the pokezz scan and then does the points.

**Comment by Linewalker** [comment]
You should remind, sniping needs a running map behind, to check, if there's a pokemon. if the map isn't available, no pokemon will be found. on the other hand, missingno's will also not befound.

**Comment by WaffleMan0310** [comment]
I have the location feeder running in the background, still no avail.

**Comment by Ferdi342** [comment]
It's working everything fine for me with @madnessxd config

**Comment by lordofthereef** [comment]
I went ahead and copied the entire contents of the p[astebin and I don't seem to be sniping successfully either...

**Comment by madnessxd** [comment]
@lordofthereef The new update made the pastebin invalid again.
